test/unit/tracks/track.test.js

```diff
@@ -21,8 +20,9 @@ TrackBaseline(Track, {
   kind: 'subtitles',
   mode: 'disabled',
   label: 'English',
-  language: 'en',
-  tech: new TechFaker()
```

Comment:
Since we need a way to cleanup this tech, we cannot just pass a `new TechFaker` into baseline. Instead I create one in every test inside `TrackBaseline` and pass it as an options.